{
  return cause.getCause() instanceof InvalidValueException;
}
